content/en/contributing.md

```diff
@@ -49,22 +48,30 @@ As website content grows, matures and adapts, it can be tricky to keep the conte
 
 - And above all, be consistent. This applies both to the words you use, and the way you use them - so things like grammar and formatting, whether you capitalise a word or not, etc.
 
-## Working with Nuxt
+### Working with Nuxt
 
 The Nuxt documentation is a great first stop for anyone new to writing technical documentation, or anyone interested in what Nuxt supports. All their guidance for writing, fetching and displaying content can be found here. It’s quick to pick up, and essential for transforming markdown files and raw text into well designed, readable guides and resources anyone can pick up.
 
 Resource: https://content.nuxtjs.org/
 
+## Contributing as a developer
```

Comment:
Can this be moved up to the top? We'll have more contributing to these projects rather than only docs.

Reply:
paragraph moved to top of file.

Reply:
@jonvnadelberg Can we move the "Contributing as a developer" section, not the Nuxt section?

Reply:
Yes.  I'll put the nuxt section back and move the contributing as a developer to the top.

Reply:
ok, the requested section has been moved.